plugin/src/main/java/com/denizenscript/denizen/events/server/ListPingScriptEvent.java

```diff
@@ -88,40 +84,31 @@ public boolean applyDetermination(ScriptPath path, ObjectTag determinationObj) {
             if (icon != null) {
                 iconCache.put(iconFile, icon);
                 event.setServerIcon(icon);
+                return true;
             }
-            return true;
-        }
-        if (determination.length() > 0 && !determineLow.equalsIgnoreCase("none")) {

```

Comment:
I was going to ask where this `none` handling went, but on closer inspection... what is this / why was it ever here? 0.o it doesn't seem to do anything in the old code.